### PR TITLE
typedocs comments

### PR DESCRIPTION
**Pull request by marcin-l-tsh**

### CODE REVIEW

File: src/ENS/ens.service.ts [hunk 1]
```diff
@@ -41,15 +41,14 @@ export class EnsService {
   ) {
     this.logger = new Logger('ENSService');
     errors.setLogLevel('error');
+
+    // Get config values from .env file
```
[the remark this answers is not in the document]
Comment (jrhender) [reply src/ENS/ens.service.ts:45]: @JGiter Should this comment appear in the docs? As it's not a typedoc comment, I wouldn't expect it to appear

**Review by jrhender — APPROVED** [review]
Looks good. All changes to actually application files seem to be formatting changes

**Comment by jrhender** [comment]
This PR looks good to me. Documentation result can be seen here: https://energy-web-foundation-did-acl-cache-server.readthedocs-hosted.com/en/myen-425-documentation-comments/
@Mani Can we merge into develop? Just want to double check as we now have auto-deploy and not sure if you want to keep current deploy due to Alpha release

File: src/ENS/ens.service.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (JGiter) [reply src/ENS/ens.service.ts:45]: I'm assuming the comment was added to make it appear in the documentation and just pointed out that it doesn't work

**Review by JGiter — APPROVED** [review]

**Pull request merged**